client/components/Header.js

```diff
@@ -1,3 +1,4 @@
+import jquery from 'jquery'
```

Comment:
And it's already included by other components..
https://github.com/MusicConnectionMachine/VisualizationG6/search?utf8=%E2%9C%93&q=jquery&type=